charts/internal/seed-controlplane/charts/cloud-controller-manager/values.yaml

```diff
@@ -1,7 +1,9 @@
 replicas: 1
+allocatorType: CloudAllocator
```

Comment:
If the cloudAllocator should not be the default for every should cluster then this line here should be changed to reflect that. I believe the simple range allocator should be enough